app/main.dev.js

```diff
@@ -75,15 +75,33 @@ const installExtensions = async () => {
   ).catch(console.log);
 };
 
+const waitForMainWindow = () => new Promise(async (resolve) => {
```

Comment:
This whole function was a doozy but it works. I had to disable linting on some lines cause I ended up in a deadlock of lint errors.